app/config/FrontendAppConfig.scala

```diff
@@ -27,6 +27,7 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
 
   private val contactHost = configuration.get[String]("contact-frontend.host")
   private val contactFormServiceIdentifier = "trustsfrontend"
+  private val agentServicesFrontendHost = configuration.get[String]("microservice.services.agent-subscription-frontend.host")
```

Comment:
Can we please load this as a `.get[Service]` to concatenate a host and port correctly from config.